filter/index.js

```diff
@@ -1,21 +1,23 @@
 function onlyEven(array) {
-  const evens = [];
-  for (let i = 0; i < array.length; i++) {
-    if (array[i] % 2 === 0) {
-      evens.push(array[i]);
-    }
-  }
-  return evens;
+  return array.filter(function(val){return val%2===0})
 }
 
 // refactor with filter!! OMG!
 
 function onlyOneWord(array) {
-  //where are the snacks?
+  return array.filter(function(word){
+    return (word.includes(' ') === false)
+    //if has a spacdont retrun it
+    //if word.split('').length <= 1
+  })
 }
 
 function positiveRowsOnly(array) {
-  //is Dan human?
+  return array.filter(function(row){
+    return row.every(function(pos) {
+      return pos >= 0;
+    });
+  });
 }
```

Comment:
Looks great!, break up your anonymous functions into fat arrow. You can eliminate the inner returns lines 17 & 18